### ISSUE TITLE
Screen isn't drawn correctly when using custom file launcher

### ISSUE THREAD

**Issue by n0ibe** [joshdick/onedark.vim#225]
- [x] I have carefully read and followed the [Installation section of the README](https://github.com/joshdick/onedark.vim#installation)
- [x] I have searched through the [closed issues](https://github.com/joshdick/onedark.vim/issues?q=is%3Aissue+is%3Aclosed) to see if there is an existing solution for my issue

### Terminal Emulator Info

Terminal emulator(s) and their version(s) used, if applicable: `alacritty 0.4.3 (c723247)`

### Output From `vim --version`

```
NVIM v0.5.0-39f802b
```

### Issue Description

Basically https://github.com/neovim/neovim/issues/12512.
